app/src/main/java/org/mozilla/fenix/settings/HomeSettingsFragment.kt

```diff
@@ -62,6 +63,15 @@ class HomeSettingsFragment : PreferenceFragmentCompat() {
             onPreferenceChangeListener = CustomizeHomeMetricsUpdater()
         }
 
+        requirePreference<Preference>(R.string.pref_key_wallpapers).apply {
+            setOnPreferenceClickListener {
+                view?.findNavController()?.navigate(
+                    HomeSettingsFragmentDirections.actionHomeSettingsFragmentToWallpaperSettingsFragment()
+                )
+                true
+            }
+        }
```

Comment:
nit: The setting should be hidden by default and only visible when `FeatureFlags.showWallpapers` is active :) 

https://github.com/mozilla-mobile/fenix/blob/dc576a2ad7bacdd28fa856c06a322916d8dc482f/app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt#L458

Reply:
Good catch, thanks!